pkgs/development/misc/resholve/README.md

```diff
@@ -102,19 +102,19 @@ trivial, so I'll also link to some real-world examples:
 ```nix
 {
   resholvedScript = resholve.writeScript "name" {
-      inputs = [ file ];
-      interpreter = "${bash}/bin/bash";
-    } ''
-      echo "Hello"
-      file .
-    '';
+    inputs = [ file ];
+    interpreter = "${bash}/bin/bash";
+  } ''
+    echo "Hello"
+    file .
+  '';
```

Comment:
Someone updated this in a recent treewide commit that focused on ensuring all Nix code in markdown blocks is valid.

Since I generate this doc from resholve and copy it in when it differs, I needed to update the code I generate from these. In the process I corrected an errant fix and picked some formatting nits.